### PR TITLE
Hw2 Brian

### PR DESCRIPTION
**Pull request by Valbjornpdx**

### CODE REVIEW

File: hw2/01-coins.js [hunk 1]
```diff
@@ -1,7 +1,44 @@
...
+    curDollar +
+    pluralize(curDollar, " dollars, ", " dollar, ") +
+    curQtr +
+    pluralize(curQtr, " quarters, ", " quarter, ") +
+    curDime +
+    pluralize(curDime, " dimes, ", " dime, ") +
+    curNickel +
+    pluralize(curNickel, " nickels, ", " nickel, ") +
+    curPenny +
+    pluralize(curPenny, " pennies.", " penny.")
```
Comment (deepika-pdx) [inline hw2/01-coins.js:40]: Hello,

The logic you used is really good. However, there is one issue that I found when I ran your code. For the below two sample cases the number of pennies it should output is slightly wrong. For example:
$4.62 ==> 4 dollars, 2 quarters, 1 dime, 0 nickel, 3 pennies.
$9.74 ==> 9 dollars, 2 quarters, 2 dimes, 0 nickel, 5 pennies.
In both the sample cases the number of pennies is greater than the actual by 1.

File: hw2/03-form.js [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+  } else {
+    console.log("Feedback: " + document.getElementById("messageBox").value);
+  }
+  console.log(
+    "Newsletter: " +
+      (document.getElementById("newsletter").checked
+        ? "Yes, I would like to join the newsletter."
+        : "No, thank you.")
+  );
+  console.groupEnd();
```
Comment (deepika-pdx) [inline hw2/03-form.js:27]: I liked your usage of console.group(). I was not aware of it. So good learning for me :)

File: hw2/03-form.html [hunk 3]
```diff
@@ -14,6 +14,30 @@
   <body class="bg-dark">
     <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
       <h1 class="mt-2 mb-4">Contact Form</h1>
+
+      <label for="nameInput" class="form-label">Name *</label>
```
Comment (deepika-pdx) [inline hw2/03-form.html:18]: Here, I think the name and email fields should be made mandatory.

**Review by deepika-pdx — APPROVED** [review]
Hello Brian,

I have added a few review comments on the respective files. Please have a look. Apart from those, everything looks good.

File: hw2/03-form.js (hunk 2 above)
Comment (aashna15sethi) [inline hw2/03-form.js:27]: this was a new learning for me about using console group. well used :D

**Review by aashna15sethi — APPROVED** [review]

File: hw2/03-form.js (hunk 2 above)
Comment (sauravpdx) [inline hw2/03-form.js:27]: Exciting to learn console group end

File: hw2/04-api.js [hunk 4]
```diff
@@ -1,7 +1,47 @@
...
+    //Beginning add one at a time via loop
+    jsonData.forEach((item) => {
+      addCountry(item);
+    });
+  } else {
+    alert("HTTP-Error: " + response.status);
+  }
+};
 
 getData(url);
```
Comment (sauravpdx) [inline hw2/04-api.js:47]: Nice clean implementation just one suggestion...  try catch block could have handled all types of exception

**Review by sauravpdx — APPROVED** [review]
Great Clean implementation loved it!

**Pull request closed without merging**